pkg/agent/openflow/pipeline.go

```diff
@@ -2853,11 +2853,20 @@ func (f *featurePodConnectivity) l3FwdFlowToExternal() binding.Flow {
 // hostBridgeLocalFlows generates the flows to match the packets forwarded between bridge local port and uplink port.
 func (f *featurePodConnectivity) hostBridgeLocalFlows() []binding.Flow {
 	cookieID := f.cookieAllocator.Request(f.category).Raw()
+	broadcastMAC, _ := net.ParseMAC("ff:ff:ff:ff:ff:ff")
```

Comment:
Can we set this at the top of this module? It is a common variable.

Reply:
updated.